backend/de.metas.business/src/main/java/de/metas/customs/CustomsInvoiceService.java

```diff
@@ -325,13 +327,15 @@ private Optional<Quantity> convertToKillogram(final Quantity qty, final ProductI
 	public String reserveDocumentNo(@NonNull final DocTypeId docTypeId)
 	{
 		final IDocumentNoBuilderFactory documentNoFactory = Services.get(IDocumentNoBuilderFactory.class);
+		final I_C_Invoice invoiceModel = InterfaceWrapperHelper.newInstance(I_C_Invoice.class);
 
 		final String documentNo = documentNoFactory.forDocType(docTypeId.getRepoId(), /* useDefiniteSequence */false)
+				.setEvaluationContext(InterfaceWrapperHelper.getEvaluatee(invoiceModel))
```

Comment:
1. why do we need `final I_C_Invoice invoiceModel`?
2. Why do we need to use an evaluation context here?
3. if evaluation context is mandatory, have you tried to use org.compiere.util.Evaluatees#empty ?

Reply:
To get the `C_Invoice` table name in `de.metas.invoice.sequence.InvoiceCountryIdProvider#computeValueInfo(org.compiere.util.Evaluatee)`, and we have the same error with use `org.compiere.util.Evaluatees#empty`:
```
Cannot get TableName from model: EMPTY
```

Reply:
what's the stacktrace?

IMHO i still believe we shall find a better solution. the one using a dummy record feels too hacky to me.